doc/api/process.md

```diff
@@ -1996,6 +1996,128 @@ Will generate an object similar to:
   tz: '2016b' }
 ```
 
+## Access Control
+
+> Stability: 1 - Experimental
+
+### process.accessControl.apply(restrictions)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `restrictions` {Object} A set of access restrictions that will be applied
+  to the current Node.js instance. The format should be the same as the one
+  returned by [`process.accessControl.getCurrent()`][]. Omitted keys will
+  default to `true`, i.e. to retaining the relevant permissions.
+
+*Warning*: This does not provide a full isolation mechanism. Child processes,
+native addons, and other communication channels may be used to circumvent
+these measures.
```

Comment:
Unfortunately, this is a dealbreaker for me. I consider best-effort security to be more dangerous than no security at all, as it gives a false sense of safety. This proposal as it currently stands is exactly that: "we provide an security API… only it's not actually secure."

Reply:
@TimothyGu Just to be clear, you *can* disable at least native addons and child processes through this API, I think this warning just really needs better wording (and maybe we should disable those two if anything else is disabled, as @jasnell suggested)…

Do you have any thoughts on that?